migrations/20170508160141_app_community_tiers.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.withSchema('app').createTable('community_tiers', function (table) {
+    table.increments('id')
+    table.bigInteger('workload_id').references('id').inTable('workload')

```

Comment:
Each of these Tier tables also needs to include the:
- TierType
- CaseType
- Points

This definition is in the WMT Schemas Changes doc under the ODP directory.